arrays/arrays.c

```diff
@@ -46,6 +90,27 @@ void destroy_array(Array *arr) {
  *****/
 void resize_array(Array *arr) {
 
+  char **arr_storage = calloc((2 * arr-> capacity), sizeof(char*));
+  for (int i=0;  i < arr->count; i++)
+  {
+    arr_storage[i] = arr->elements[i];
+  }
+  for(int i = 0; i < arr->count; i++){
```

Comment:
It looks like this for loop is freeing the strings the old elements are pointing to. It looks like it might not be needed.